### ISSUE TITLE
Does not bring "dependencies" into the migrated package.json

### ISSUE THREAD

**Issue by MichalBryxi** [NullVoxPopuli/ember-addon-migrator#21]
My original [/package.json](https://gitlab.com/michal-bryxi/open-source/ember-intl-changeset-validations/-/merge_requests/123/diffs#7030d0b2f71b999ff89a343de08c414af32fc93a_37_11) in the root of the repo has:

```json
  "dependencies": {
    "ember-changeset-validations": "^3.16.0",
    "ember-cli-babel": "7.26.11",
    "ember-cli-htmlbars": "6.0.1",
    "ember-intl": "5.7.2"
  },
```

But after running `npx ember-addon-migrator@latest` the [/ember-intl-changeset-validations/package.json](https://gitlab.com/michal-bryxi/open-source/ember-intl-changeset-validations/-/merge_requests/123/diffs#4f5849148e3f9910dbb911c8dd29f2640529d4d9_0_32) has only:

```json
  "dependencies": {
    "@embroider/addon-shim": "^1.2.0"
  },
```

Should the migration bring the entries in the original `dependencies` key along?

**Comment by NullVoxPopuli** [comment]
It shouldn't. At least not as is. 
Managing dependencies isn't implemented atm, but,
The ember-cli* packages are no longer needed.
ember-intl should probs be a peerDependency, and changeset validations makes sense as either dep or peerDep.

Moving deps over could probably be implemented with only a couple rules for filtering stuff out.

**Comment by NullVoxPopuli** [comment]
This is implemented in 1.6.0
